firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -6,22 +8,30 @@ You may pause and resume the script from the last batch at any point.
 
 #### Important notes
 
-+   Run the script over the entire collection **_after_** installing the Export Collections to BigQuery extension; otherwise the writes to your database during the import might not be exported to the dataset.
++   You must run the script over the entire collection **_after_** installing the Export Collections to BigQuery extension; otherwise the writes to your database during the import might not be exported to the dataset.
 +   The import script can take up to _O(collection size)_ time to finish. If your collection is large, you might want to consider [loading data from a Cloud Firestore export into BigQuery](https://cloud.google.com/bigquery/docs/loading-data-cloud-firestore).
-+   You will see redundant rows in your raw changelog table:
++   You will see redundant rows in your raw changelog table if either of the following happen:
 
     +   If document changes occur in the time between installing the extension and running this import script.
     +   If you run the import script multiple times over the same collection.
 
 ### Install and run the script
 
-This import script uses several values from your installation of the extension:
+The import script uses several values from your installation of the extension:
 
 +   `${PROJECT_ID}`: the project ID for the Firebase project in which you installed the extension
 +   `${COLLECTION_PATH}`: the collection path that you specified during extension installation
 +   `${DATASET_ID}`: the ID that you specified for your dataset during extension installation
 
-1.  Run `npx @firebaseextensions/fs-bq-import-collection`.
+Install the import script using `npm` (the Node Package Manager).
+
+1.  Make sure that you have [Node.js](https://www.nodejs.org/) installed in order to access the `npm` command tools.
+
+1.  Install the import script via `npm` by running the following command:
+
+    ```
+    npx @firebaseextensions/fs-bq-import-collection
```

Comment:
Need to also install npx, should add this line above line 33:

`npm install -g npx`

Reply:
npm comes packaged with npm version 5.2+: https://blog.scottlogic.com/2018/04/05/npx-the-npm-package-runner.html

But good to install it explicitly in case the user has a lower version of npm.

Reply:
Maybe we can add a [callout](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes) right below the step, like this:

...

> If you're on a very old version of npm (lower than 5.2), you'll need to install npx first: `npm install --global npx`

...

I think it is safe to just have a minor callout because the oldest node LTS start date is [October 2017](https://nodejs.org/en/about/releases/) and npm 5.2 came out in [July 2017](https://github.com/npm/npm/releases/tag/v5.2.0), so doing the install is hopefully unnecessary for most.

Reply:
PTAL?

Reply:
LGTM!

Reply:
PR just approved by @laurenzlong.  Resolving this thread.  Thx!